test/lashup_kv_aae_SUITE.erl

```diff
@@ -0,0 +1,168 @@
+-module(lashup_kv_aae_SUITE).
```

Comment:
@GoelDeepak Could you please use 4 spaces for indentation in all new modules?

Reply:
The module is a copy-paste from `lashup_hyparview_SUITE`, would it be possible to add `lashup_kv_aae_test` to there?

1. create new node
2. wait for aae
3. stop it
4. repeat [1-3]

Reply:
I intentionally kept it separate so as not to mix with Hyparview tests. It is easier to spot the tests if they are separated at functionality level. I understand that it is a repetation of the code but choose clarity over repetation as it is a test code.

Reply:
JIRA: [DCOS_OSS-3605](https://jira.mesosphere.com/browse/DCOS_OSS-3605) Improve common tests for lashup (Hyparview and AAE)